### PR TITLE
Make callback example link relative

### PR DESCRIPTION
**Pull request by MikeRalphson**
Saves having to remember to update it later.

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -1820,7 +1820,7 @@ The key value used to identify the callback object is an expression, evaluated a
 ##### Patterned Fields
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="callbackExpression"></a>{expression} | [Path Item Object](#pathItemObject) | A Path Item Object used to define a callback request and expected responses.  A [complete example](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/examples/v3.0/callback-example.yaml) is available.
+<a name="callbackExpression"></a>{expression} | [Path Item Object](#pathItemObject) | A Path Item Object used to define a callback request and expected responses.  A [complete example](/examples/v3.0/callback-example.yaml) is available.
```
Comment (darrelmiller) [inline versions/3.0.md]: Doesn't that need a `..` at the front of that URL?

Comment (MikeRalphson) [inline versions/3.0.md]: Seems to work ok in my fork, i.e. it's a repo-root relative url.

Comment (ePaul) [inline versions/3.0.md]: I guess the difference is that with the `..` it points to the example in the current branch, without it points to master (where this file doesn't exist yet).

Comment (MikeRalphson) [inline versions/3.0.md]: The link in my fork points to https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/examples/v3.0/callback-example.yaml - not the master branch? Viewing the changes I see a link to the [commit blob](https://github.com/MikeRalphson/OpenAPI-Specification/blob/7dd9891b20ceb7232c9704dbc7b3f7bbff30204e/examples/v3.0/callback-example.yaml)

Comment (ePaul) [inline versions/3.0.md]: Hmm, I looked at it in the [differences view](https://github.com/OAI/OpenAPI-Specification/pull/1223/files?short_path=4a97aef#diff-4a97aef5bba9ddb7fd8609f48f4bdc30), and there it pointed to https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v3.0/callback-example.yaml. Strangely, if I open the same address again in a new browser tab, it now points to https://github.com/OAI/OpenAPI-Specification/blob/rel_link/examples/v3.0/callback-example.yaml.  (Both links give 404s currently, the second one uses your branch name with the main fork's repository name.)

I guess Github it not that predictable with the link expansion.

Comment (MikeRalphson) [inline versions/3.0.md]: Thanks @ePaul for looking at it again. Let's try it with the ..

**Pull request merged**